### PR TITLE
DAG-1186 Fix mapping of TaskStats fields

### PR DESCRIPTION
**Pull request by carlos-anaya**

### CODE REVIEW

File: src/evergreen/stats.py [hunk 1]
```diff
@@ -39,10 +39,10 @@ class TaskStats(_BaseEvergreenObject):
     task_name = evg_attrib("task_name")
```
Comment (carlos-anaya) [inline src/evergreen/stats.py:39]: The [TaskStats](https://github.com/evergreen-ci/evergreen/blob/f10690476e6d3b0a69b2d9432f513eb22a7e23bc/model/stats/query.go#L278-L293) response doesn't have a `test_file` field. Should we remove it?

Comment (dbradf) [reply src/evergreen/stats.py:39]: Yes, let's remove it.

**Review by dbradf — COMMENTED** [review]
Could you update the version in the `pyproject.toml` file?

**Review by dbradf — APPROVED** [review]

**Comment by carlos-anaya** [comment]
evergreen merge

**Pull request merged**